Microsoft.NET.Build.Containers.IntegrationTests/FullFramework/CreateNewImageToolTaskTests.cs

```diff
@@ -228,6 +228,14 @@ public void GenerateCommandLineCommands_Labels()
                                       --labels "NoValue=\"\"" "Valid1=\"Val1\"" "Valid12=\"Val2\"" "Valid12=\"\""
                                       """, args);
         Assert.Equal("Items 'Labels' contain empty item(s) which will be ignored.", Assert.Single(warnings));
+
```

Comment:
added calling the real containerize to check for parse error for non-trivial cases